src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
It was a compilation error, not a warning. I've disabled the error on warning on the config file, to ease myself with the porting effort, but it wasn't liking it at all. I will post the full error if you need it.